test/flag_shih_tzu_test.rb

```diff
@@ -1250,8 +1250,8 @@ def test_should_define_bang_methods
   end
 
   def test_should_return_a_sql_set_method_for_flag
-    assert_equal "flags = flags | 1",  Spaceship.send( :sql_set_for_flag, :warpdrive, 'flags', true)
-    assert_equal "flags = flags & ~1", Spaceship.send( :sql_set_for_flag, :warpdrive, 'flags', false)
+    assert_equal "flags = flags | 1",  Spaceship.send( :sql_set_for_flag, :warpdrive, "flags", true)
```

Comment:
Line is too long. [100/80]<br>Unnecessary spacing detected.<br>Space inside parentheses detected.